server/events/command_runner.go

```diff
@@ -329,6 +349,41 @@ func (c *DefaultCommandRunner) automerge(ctx *CommandContext, pullStatus models.
 	}
 }
 
+func (c *DefaultCommandRunner) runProjectCmdsParallel(cmds []models.ProjectCommandContext, cmdName models.CommandName) CommandResult {
+	var results []models.ProjectResult
+	mux := &sync.Mutex{}
+
+	wg := sizedwaitgroup.New(c.ParallelPlansPoolSize)
+	for _, pCmd := range cmds {
+		pCmd := pCmd
+		var execute func()
+		wg.Add()
+
+		switch cmdName {
+		case models.PlanCommand:
+			execute = func() {
+				defer wg.Done()
+				res := c.ProjectCommandRunner.Plan(pCmd)
+				mux.Lock()
+				results = append(results, res)
+				mux.Unlock()
+			}
+		case models.ApplyCommand:
```

Comment:
We're not running apply's in parallel right?

Reply:
Nope! Too many unknowns so I left this out

Reply:
But it's still in here... unless I'm missing something?

Reply:
Oops you're right, this branch is never run because of the check here: https://github.com/runatlantis/atlantis/pull/926/files#diff-110138e4d98a96febb9cd1393b609896R262

I'll change it to throw an exception instead, in case it's called elsewhere in the future

Reply:
I've added a flag to run applies in parallel, default false